src/Raven.Server/Raven.Server.csproj

```diff
@@ -240,4 +240,8 @@
     <ConcurrentGarbageCollection>false</ConcurrentGarbageCollection>
     <RetainVMGarbageCollection>false</RetainVMGarbageCollection>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(AllowEncryptedWithoutSsl)' == 'True'">
+    <DefineConstants>$(DefineConstants);ALLOW_ENCRYPTED_SSL_ONLY</DefineConstants>
```

Comment:
let's be consistent `ALLOW_ENCRYPTED_WITHOUT_SSL`

Reply:
Done